### PR TITLE
feat: include stream in db, rework blacklisting

### PR DESCRIPTION
**Pull request by Gaisberg**

### CODE REVIEW

File: src/program/state_transition.py [hunk 1]
```diff
@@ -27,6 +27,13 @@ def process_event(existing_item: MediaItem | None, emitted_by: Service, item: Me
         if existing_item and not TraktIndexer.should_submit(existing_item):
             return no_further_processing
         return None, next_service, [item]
+    
+    elif item.state == States.Indexed and len(item.streams) > 0:
```
Comment (dreulavelle) [inline src/program/state_transition.py:31]: this section needs to go

File: src/utils/request.py [hunk 2]
```diff
@@ -95,13 +95,6 @@ def _make_request(
     if retry_if_failed:
         session.mount("http://", _adapter)
         session.mount("https://", _adapter)
-    headers = {
```
Comment (dreulavelle) [inline src/utils/request.py:98]: what made you get rid of this? Just curious

Comment (Gaisberg) [reply src/utils/request.py:98]: Torbox api wasnt responding to those headers very well. If a service needs them then lets implement them st that level?

File: src/program/state_transition.py (hunk 1 above)
Comment (Gaisberg) [reply src/program/state_transition.py]: My bad, left that in there.

File: src/utils/request.py (hunk 2 above)
Comment (dreulavelle) [reply src/utils/request.py:98]: Sure. I wasn't sure how the curl's from that list were gonna behave too

**Pull request merged**